src/helpers/saveFormat.ts

```diff
@@ -38,6 +41,8 @@ export function makeSave(settings: Partial<IState['settings']>) {
   output += `:P${pornList.reduce((acc, porn) => `${acc}${acc ? ',' : ''}${encodePorn(porn)}`, '')}`
   output += `:E{${eventList.reduce((acc, eventId) => `${acc}${acc ? ',' : ''}${eventId}`, '')}}`
   output += `:H${hypnoMode}`
+  output += `:PG${playerMode.gender}`
+  output += `:PP${playerMode.parts}`
```

Comment:
I'm so sorry that my save format is such a mess haha Good job getting this working!